app/views/unattended/provisioning_templates/PXEGrub/pxegrub_default_local_boot.erb

```diff
@@ -23,4 +23,4 @@ description: |
 default=<%= default_item %>
 timeout=20
 
-<%= snippet "pxegrub_chainload", variables: {paths: paths} %>
+<%= snippet([['custom pxegrub_chainload', 'pxegrub_chainload']], variables: {paths: paths}) %>
```

Comment:
Why is this a nested array?

Reply:
So using a very simple template to test this

```
<%= snippet_if_exists(["asdf", "Kickstart default custom postnochroot"]) -%>
adsf
this shosuld be outputting
```

when it is a single value ie `Kickstart default custom postnochroot` it is still happy and works as you would expect.  Adding the second value, does not exist, of `asdf`, it results in the error `There was an error rendering the template: no implicit conversion of Symbol into Integer`.  Shifting to `[[` from `[` gets back to working.

Quickly trying again with a variable

```
<% myvar = ["asdf", "Kickstart default custom postnochroot"] %>

<% myvar.each do |mv| %>
    <%= mv %>
<% end %>

<%= snippet_if_exists(myvar) %>
```

this seems to behave like an array in the loop and work when passed to the function.  However utilizing `[` directly in the function call results in the symbol error mentioned above.  I didn't like the idea of adding a var line just to be able to specify an array of templates and at some point tried the `[[` which brings us to here.  I'd absolutely like to understand this because I agree, as written it looks strange and is likely to cause confusion long term.  I'm also not sure what to search for at this point to try and unravel this further.  I think all of the examples found while searching showed it as a two step, declare and then pass...